libvncserver/ws_decode.h

```diff
@@ -0,0 +1,151 @@
+#ifndef _WS_DECODE_H_
```

Comment:
Something for a TODO list at the Moment. But in general, no, it would very probably not be a lot of work.

Maybe it would also not be the worst idea ever to consider using some library to implement the websockets part.